docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_toggleswitch.htm

```diff
@@ -13,9 +13,7 @@ <h1>Toggle Switch</h1>
 
 <p>The toggle switch component shows on the screen a 2-state switch.</p>
 
-<video autoplay muted loop controls>
-	<source src="../../images/toggleswitch.mp4">
-</video>
+<img src="../../images/Toggleswitch.png" alt="photo of Toggleswitch element in Tizen">
```

Comment:
Are you going to remove the mp4 file and use the png file instead?
If so, how about removing not used toggleswitch.mp4?

Reply:
Done